.github/CODE_OF_CONDUCT.md

```diff
@@ -3,7 +3,7 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
+contributors and maintainers pledge to make participation in our project and
```

Comment:
We have the same pledge across our OSS repos, so if we change it here we should probably change it everywhere.